Android/ZensoftHRCRM/app/src/main/java/com/erkprog/zensofthrcrm/CRMApplication.java

```diff
@@ -1,13 +1,17 @@
 package com.erkprog.zensofthrcrm;
 
 import android.app.Application;
+import android.content.res.Resources;
 
 public class CRMApplication extends Application {
 
+  public static Resources resources;
```

Comment:
No need to make resources as a singleton. Use context for getting each string resource